content/en/infrastructure/process/_index.md

```diff
@@ -207,7 +207,7 @@ For more information about installing the Datadog Agent with AWS ECS Fargate, se
 
 ### I/O stats
 
-I/O and open files stats can be collected by the Datadog system-probe, which runs with elevated privileges. To enable the process module of the system-probe, use the following configuration:
+I/O and open files stats can be collected by the Datadog system-probe, which runs with elevated privileges. To collect these stats, use the following configuration to enable the process module of the system-probe:
```

Comment:
Added some clarification here that enabling the process module enables I/O stats collection, as otherwise it seemed a bit disjointed